client/blocks/login/two-factor-authentication/verification-code-form.scss

```
@@ -100,3 +113,14 @@
 .security-key-form__add-wait-for-key-heading {
 	font-weight: 600;
 }
+
+div.two-factor-auth-enabled:not(.woo) {
```

Let's avoid this if we can. The design should be the same between Woo and .com, except for the font and color change.

---

I completely agree. It would be great to narrow the gap between WPCOM and Woo designs.

---

We can avoid using the woo specific classes and the !important here by increasing the specificity:
```suggestion
.layout .wp-login__main .login .login__form-header {
	width: var( --login-container-width );
	margin: 0 auto;
	padding: 0 24px;
	text-align: left;
	line-height: 40px;
}
```
> It would be great to narrow the gap between WPCOM and Woo designs.

To do that, I think we should only add the properties here that apply to both Woo and WP.com versions. We should specify the different fonts/colors in the respective heading rules.

---

Yeah. It would work great. I will update the CSS selector to increase the specificity.